### DISCUSSION TITLE
Notes location on windows

### DISCUSSION THREAD

**Discussion by giuseppepagnoni** [TriliumNext/Trilium#3216] [Q&A] [answered]
Where does Trilium store notes on windows? Is it a single file or multiple file? Can I select a custom location (e.g. a folder in onedrive)?
I would like to bakup them regularly to avoid data loss.

Thanks!

**Comment by sigaloid** [comment, marked as answer]
Your notes are stored in a single folder: https://github.com/zadam/trilium/wiki/Document

It's not recommended to use onedrive to sync the main file, but you can enable [Trilium backups](https://github.com/zadam/trilium/wiki/Backup) and enable backing up of that folder - backing up the main file can cause conflicts because Onedrive isn't used to syncing files that change while in use, usually.

**Reply by giuseppepagnoni** [reply]
Fantastic, thanks!

**Comment by zadam** [comment]
BTW, you can see the data storage location in the About dialog:

![image](https://user-images.githubusercontent.com/617641/197340602-b767923b-0f3a-406f-8c4b-09076ebc6417.png)
